packages/flutter/test/widget/navigator_test.dart

```diff
@@ -40,6 +40,28 @@ class SecondComponentState extends State<SecondComponent> {
   }
 }
 
+class ThirdComponent extends StatelessComponent {
+  static final Key targetKey = new Key('foo');
```

Comment:
It's probably mildly better to have this component take a key as an argument that it passes to its superclass and then for the caller below to create and track the key rather than having this key in the global scope.